### PR TITLE
Update error summary message for o-forms

### PR DESCRIPTION
**Pull request by JuanTincho**
## Description

- feat: update o-colors to fix bg color
- feat: add custom optional message to error summary
- feat: add alert icon to error summary

## Ticket:
https://financialtimes.atlassian.net/browse/ACQ-1918

### CODE REVIEW

File: components/o-forms/src/scss/shared/_error-summary.scss [hunk 1]
```diff
@@ -1,15 +1,60 @@
+$o-forms-error-summary-foreground-color: oColorsMix(
+	'black',
+	_oFormsGet('invalid-base'),
+	$percentage: 12.5
+);
+$o-forms-error-summary-background-color: oColorsMix(
```
Comment (notlee) [inline components/o-forms/src/scss/shared/_error-summary.scss:6]: Please prepend these variable names with an underscore `$_o-forms...` to indicate that they are private variables. Other than that this looks good to me @JuanTincho! :)

**Review by notlee — CHANGES_REQUESTED** [review]

File: components/o-forms/src/scss/shared/_error-summary.scss (hunk 1 above)
Comment (JuanTincho) [reply components/o-forms/src/scss/shared/_error-summary.scss]: Thanks! Fixed!

File: components/o-forms/src/scss/shared/_error-summary.scss [hunk 2]
```diff
@@ -1,15 +1,60 @@
...
+);
+
+$_o-forms-alert-icon-size: 2rem;
+$_o-forms-alert-icon-inbuilt-space: $_o-forms-alert-icon-size * 0.25;
+
+// Icon svgs are drawn to a 40px grid with 10px whitespace either side.
+// This is what the size of the alert icon would be without the whitespace.
+$true-alert-icon-size: div($_o-forms-alert-icon-size, 2);
+$alert-icon-left-padding: $_o-forms-spacing-four;
+$alert-icon-right-padding: $_o-forms-spacing-two;
```
Comment (notlee) [inline components/o-forms/src/scss/shared/_error-summary.scss:18]: As we haven't switched to use dart sass modules yet these variables are global and could clash with those from different components / projects. You'll need to prepend them with `$_o-forms-` like the others or scope them inside the `_oFormsErrorSummary` mixin

Comment (JuanTincho) [reply components/o-forms/src/scss/shared/_error-summary.scss]: Oh! that makes sense. Sorry!  Pushed it again 😄

**Review by notlee — APPROVED** [review]

**Pull request merged**